### ISSUE TITLE
"Memory" Card Game Branch

### ISSUE THREAD

**Issue by MattSurabian** [boxart/boxart-boiler#16]
Create demo games as branches that can be deployed via Github Pages. The first one we'll do is a card matching memory style game. This game should be usable via screen reader.
